airbyte-integrations/bases/base-normalization/unit_tests/test_table_name_registry.py

```diff
@@ -129,26 +130,29 @@ def read_json(input_path: str, apply_function=(lambda x: x)):
 
 
 @pytest.mark.parametrize(
-    "json_path, expected_postgres, expected_bigquery",
+    "json_path, expected_postgres, expected_bigquery, expected_oracle",
     [
         (
             ["parent", "child"],
             "parent_child",
             "parent_child",
+            "PARE__HILD"
         ),
         (
             ["The parent stream has a nested column with a", "short_substream_name"],
             "the_parent_stream_ha___short_substream_name",
             "The_parent_stream_has_a_nested_column_with_a_short_substream_name",
+            "THE___NAME",
         ),
         (
             ["The parent stream has a nested column with a", "substream with a rather long name"],
             "the_parent_stream_ha__th_a_rather_long_name",
             "The_parent_stream_has_a_nested_column_with_a_substream_with_a_rather_long_name",
+            "THE___NAME"
```

Comment:
@ChristopheDuong can you validate this, it's weird whats going on

Reply:
30 characters limit - 12 reserved for dbt - 8 for airbyte = table name of 10 characters long...